### DISCUSSION TITLE
Disabling a Primary Category on the platform

### DISCUSSION THREAD

**Discussion by Xitija** [Sunbird-Knowlg/Community#86] [Q&A] [answered]
Hello All,

I want to delete/disable the primary categories on the platform.
How can it be done? Is there any API that can perform this task ?

Please guide me here .

cc: @komalm @Renuka-S

**Comment by AmiableAnil** [comment, marked as answer]
@Xitija Deleting the primary categories is not possible through APIs. You can delete the ObjectCategory and ObjectCategoryDefinition directly from DBs (neo4j and cassandra).

**Reply by Krishnaj20** [reply]
@Xitija - let us know if your query is resolved so that we mark this thread as answered.

@pallakartheekreddy

**Reply by Xitija** [reply]
Yes you can mark this as answerd.
